src/app/components/cms-widget/cms-display.jsx

```diff
@@ -236,25 +259,31 @@ function ChplCmsDisplay() {
       { idAnalysis.missingAnd?.length > 0
         && (
           <>
-            <Typography>Please select a product or products that contain the following criteria:</Typography>
-            <List>
-              { idAnalysis.missingAnd.map((criterion) => <ListItem key={criterion}>{ criterion }</ListItem>)}
-            </List>
+            <Divider />
+            <Box mt={2}>
+              <Typography variant="body2">Please select a product or products that contain the following criteria:</Typography>
+              <Typography variant="body2">
```

Comment:
I think this has to be a `<List>` element, in order to have `<ListItem>` elements under it. Otherwise you end up with

```
<p>
   <li>
   <li>
</p>
```

which isn't valid HTML